updates/2024-02-12-editorial-board-update.md

```diff
@@ -0,0 +1,119 @@
+# Meeting Notes: Python Docs Editorial Board
+
+## February 12, 2024
+
+## Attendees
+
+Mariatta, Ned, Joanna, Guido
+
+
+## Agenda
+
+1. What project do we want to deliver first? Here are some ideas to start the discussion:
+
+a. Define scope and outline for new intro tutorial
+b. Define scope and outline for new landing page
+c. Inventory/audit a section of the docs
+d. Personas (see #3 below)
```

Comment:
Do this and the following sublists need to be indented?
I'm also not sure MD understands letters.

```suggestion
1. What project do we want to deliver first? Here are some ideas to start the discussion:

   a. Define scope and outline for new intro tutorial
   b. Define scope and outline for new landing page
   c. Inventory/audit a section of the docs
   d. Personas (see #3 below)
```